app_flutter/lib/state/build.dart

```diff
@@ -41,7 +41,10 @@ class BuildState extends ChangeNotifier {
 
   /// Whether or not flutter/flutter currently passes tests.
   bool get isTreeBuilding => _isTreeBuilding;
-  bool _isTreeBuilding;
+  bool _isTreeBuilding = true;
```

Comment:
Should this be left null?

Reply:
Wouldn't this remove the loading animation for the tree status?

Would explicitly checking `_isTreeBuilding == false` work for the use case below?

Reply:
Ahh, I didn't consider that.  Yes, I can check below instead.